Mobile Buy SDK/Mobile Buy SDK/Models/BUYCheckout.h

```diff
@@ -225,7 +220,7 @@
 @property (nonatomic, copy, readonly) NSString *sourceName;
 
 /**
- *  The unique identifier for the source: the channelId
+ *  The unique identifier for the source: the appId
  */
 @property (nonatomic, copy, readonly) NSString *sourceIdentifier;
```

Comment:
What were you sending for source_name, the idea is to remove this field in the long run.